### ISSUE TITLE
[Accessibility]: 

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41682]
### Audit type

Screen reader navigation

### Problem description

![PixelSnap 2022-09-15 at 12 21 56](https://user-images.githubusercontent.com/1855233/190470004-4046ace9-a0b7-4983-8491-ebe356d17ee7.png)

When focusing on the "three dots" menu button, voice-over "Dashboard options dashboard options menu popup button"

### Expected behavior

Should only say "Dashboard options menu popup button"

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
